x-pack/platform/plugins/private/upgrade_assistant/public/application/components/es_deprecations/deprecation_types/indices/flyout/steps/details/reindex_details_step.tsx

```diff
@@ -40,6 +40,30 @@ import { IndexClosedParagraph } from '../index_closed_paragraph';
 
 const ML_ANOMALIES_PREFIX = '.ml-anomalies-';
 
+const FollowerIndexCallout = () => (
+  <>
+    <EuiCallOut
+      title={
+        <FormattedMessage
+          id="xpack.upgradeAssistant.esDeprecations.indices.indexFlyout.detailsStep.followerIndexTitle"
+          defaultMessage="Unfollow leader index is recomended"
+        />
+      }
+      color="primary"
+      iconType="iInCircle"
+      data-test-subj="followerIndexCallout"
+    >
+      <p>
+        <FormattedMessage
+          id="xpack.upgradeAssistant.esDeprecations.indices.indexFlyout.detailsStep.followerIndexText"
+          defaultMessage="This index is a cross-cluster replication follower index. We recomend you to unfollow the leader index."
```

Comment:
This paragraph alone wasn't sufficient for me to understand why I'd need to do this (and what the next steps were).  If we have room, i think it would be useful to convey more of the information from the issue description. For example:

```suggestion
          defaultMessage="This index is a cross-cluster replication follower index, which should not be reindexed. You can mark it as read-only or terminate the replication and convert it to a standard index."
```

In this case I think it would also be helpful to refer to the documentation about how to terminate replication. The aforementioned docs page doesn't exist in the doc link service yet, so it would needed to be added in https://github.com/elastic/kibana/blob/main/src/platform/packages/shared/kbn-doc-links/src/get_doc_links.ts